app/javascript/pages/components/partials/CallToAction.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CallToAction = ({ link, text }) => (
+  <div className="cta__wrapper">
```

Comment:
While cta is shorter I would recommend renaming to `call-to-action__wrapper` et al. to make it more meaningful.